setup.py

```diff
@@ -48,7 +50,25 @@ def find_version(*file_paths):
 
 if sys.version_info[:2] < (3, 4):
     tests_require.append('mock >= 1.0.1')
-    install_requires.append('enum34 >= 1.0.4, < 2')
+
+extras_require = {
+    ':python_version < "3.4"': ['enum34 >= 1.0.4, < 2']
+}
+
+
+try:
+    if 'bdist_wheel' not in sys.argv:
+        for key, value in extras_require.items():
+            if key.startswith(':') and pkg_resources.evaluate_marker(key[1:]):
+                install_requires.extend(value)
+except Exception:
+    logging.getLogger(__name__).exception(
+        'Something went wrong calculating platform specific dependencies, so '
+        "you're getting them all!"
+    )
+    for key, value in extras_require.items():
+        if key.startswith(':'):
+            install_requires.extend(value)
```

Comment:
@kevin1024 and I worked this out over a few iterations of users with odd Python installs https://github.com/kevin1024/vcrpy/blob/master/setup.py#L37-L49